Assets/Scripts/Camera/PlayerCamera.cs

```diff
@@ -8,19 +7,27 @@ public class PlayerCamera : MonoBehaviour, DefaultInputActions.IPlayerCameraActi
     {
         public float sensX;
         public float sensY;
-        public float cameraChangeY;
-        public float cameraChangeZ;
+        public float zoomSensitivityMultiplier;
+        public Vector3 thirdPersonCameraOffset;
         public float rotationClampTop;
         public float rotationClampBottom;
         public float wallCollisionOffset;
+        public float zoomMagnification;
+        public float zoomSpeed;
         public Transform playerObject;
         public Transform cameraHolder;
+
+        private Camera cam;
         private DefaultInputActions actions;
         private float xRotation;
         private float yRotation;
         private float xInput;
         private float yInput;
-        private bool isFirstPerson = true;
+        private Vector3 cameraOffset = Vector3.zero;
+        public bool isFirstPerson = true;
+        public bool isZoomed = false;
```

Comment:
Move above private fields. 

Should this be public? Maybe change to property: 

```suggestion
        public bool isFirstPerson {get; private set} = true;
        public bool isZoomed {get; private set; }= false;
```